spec/features/user/can_create_a_trip_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
This is super nitpicky, but do we want to specify that it's a driver that can create a trip, either by putting it in a driver folder or adding that to the beginning of the name?